File: problemas/1100/1132.dart
import 'dart:io';
import 'dart:math';

void main() {
  final x = int.parse(stdin.readLineSync());
  final y = int.parse(stdin.readLineSync());
  final menor = min(x, y);
  final maior = max(x, y);
  int soma = 0;

  for (int i = menor; i <= maior; i++) {
    if (i % 13 != 0) {
      soma += i;
    }
  }

  stdout.writeln(soma);
}

File: problemas/1100/1132.js
const [x, y] = require('fs').readFileSync('/dev/stdin', 'utf8').split('\n');
const [menor, maior] = [Math.min(x, y), Math.max(x, y)];
let soma = 0;

for (let i = menor; i <= maior; i++) {
  if (i % 13 != 0) {
    soma += i;
  }
}

console.log(soma);